app/src/sharedTest/java/org/oppia/android/app/story/StoryFragmentTest.kt

```diff
@@ -499,7 +499,7 @@ class StoryFragmentTest {
   }
 
   private fun setStoryPartialProgressForFractions() {
-    storyProgressTestHelper.markPartialStoryProgressForFractions(
+    storyProgressTestHelper.markChapDoneFrac0Story0Exp0(
```

Comment:
one test in this file is wrong - `testStoryFragment_configChange_chapterMissingPrerequisiteThumbnailIsBlurred`

We don't show the image while the orientation is Landscape, so we cannot test isBlurred in the landscape. And the reason it is passing on robolectric is the orientation is not happening in robolectric which is getting fixed in one of Ben's PR. So, we should remove the test.

Reply:
This test is not related to this PR. We can create a issue on this.

Reply:
Could you confirm this and file an issue for this if exist now too?

Reply:
This is failing on expresso. Could you please file an issue, as you have better context on this.